bugbug/utils.py

```diff
@@ -280,14 +294,43 @@ def create_tar_zst(path: str) -> None:
     if not os.path.exists(inner_path):
         raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), inner_path)
 
-    subprocess.run(["tar", "-I", "zstdmt", "-cf", path, inner_path], check=True)
+    try:
+        subprocess.run(
+            ["tar", "-I", "zstdmt", "-cf", path, inner_path],
+            check=True,
+            stderr=subprocess.PIPE,
+        )
+    except subprocess.CalledProcessError as error:
+        if (
+            error.returncode == 2
+            and "zstdmt: not found" in str(error.stderr, "utf-8").strip()
```

Comment:
Decoding `stderr` from bytes to a string improves search accuracy because the search query is a string. Using UTF-8 encoding also ensures consistency across different platforms, as default encodings may vary. 🌝